fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransaction.java

```diff
@@ -458,6 +458,16 @@ public void reverse(final ExternalId reversalExternalId) {
         this.reversalExternalId = reversalExternalId;
     }
 
+    public void reverseWithExternalIdHandling(final boolean isReverseReplayed) {
```

Comment:
This is not needed! I dont wanna move "reverse-replay" logic into the LoanTransaction entity...